ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: openshift-upgrade-qe-test-cert-rotation
+  steps:
+  - ref: cucushift-upgrade-helper-service-build-info
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-healthcheck
+  - ref: cucushift-upgrade-toimage
+  - ref: openshift-extended-upgrade-post
+  - ref: openshift-oc-certrotation
```

Comment:
Refer to dev's [pr](https://github.com/openshift/release/pull/40388/files), certrotation ran before e2e test. Should we locate `openshift-oc-certrotation` before `openshift-extended-upgrade-post`? cc feature owner @sunzhaohua2 @xingxingxia @rioliu-rh to confirm.

Reply:
Agree.

Reply:
Thanks, updated